messagelib/contracts/uln/uln301/AddressSizeConfig.sol

```diff
@@ -10,12 +10,12 @@ abstract contract AddressSizeConfig is Ownable {
 
     event AddressSizeSet(uint16 eid, uint256 size);
 
-    error InvalidAddressSize();
-    error AddressSizeAlreadySet();
+    error AddressSizeConfig_InvalidAddressSize();
+    error AddressSizeConfig_AddressSizeAlreadySet();
```

Comment:
If `AddressSizeConfig` is imported by the send and receive libs whose errors are in the `LZ_` domain, those might also need to be?